### PR TITLE
Prevent duplicate `jsbundling-rails` in Gemfile

### PR DESCRIPTION
**Pull request by andyw8**
Similar to https://github.com/excid3/jumpstart/pull/174, I was seeing a warning for `jsbundling-rails`.

### CODE REVIEW

**Comment by excid3** [comment]
Lets just apply this to all gem dependencies.

File: template.rb [hunk 1]
```diff
@@ -222,6 +219,10 @@ def add_esbuild_script
   end
 end
 
+def gem_exists?(name)
```
Comment (excid3) [inline template.rb:222]: Let's do an `add_gem` method that safely adds a gem.

Something like this:
```ruby
def add_gem(name, *options)
  gem(name, *options) unless gem_exists?(name)
end
```

**Review by excid3 — CHANGES_REQUESTED** [review]
We're going to eventually need to deal with all the gems I imagine, so might as well do it all at once. 👍

File: template.rb (hunk 1 above)
Comment (andyw8) [reply template.rb:226]: 👍 Updated in a49f4f29f11cee913aa19c9f5b718a5b79a85506

**Review by excid3 — APPROVED** [review]
Thanks @andyw8!

**Pull request merged**